### PR TITLE
Update test_python_function_operator.py

### PR DESCRIPTION
**Pull request by kychennv**
add test_python_operator_rotate, it could be passed when https://github.com/NVIDIA/DALI/pull/524/file fixed

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: dali/pipeline/operators/displacement/rotate.h [hunk 1]
```diff
@@ -33,9 +33,9 @@ class RotateAugment : public WarpAffineAugment {
     float angle = spec.GetArgument<float>("angle", ws, index);
     float angle_rad = angle * (M_PI / 180.0);
     p->matrix[0] = std::cos(angle_rad);
-    p->matrix[1] = std::sin(angle_rad);
```
Comment (jantonguirao) [inline dali/pipeline/operators/displacement/rotate.h]: isn't this changing the behavior?

File: dali/test/python/test_operator_python_function.py [hunk 2]
```diff
@@ -172,9 +180,56 @@ def test_python_operator_flip():
         for i in range(len(numpy_output)):
             assert numpy.array_equal(numpy_output.at(i), dali_output.at(i))
 
+
+class RotatePipeline(CommonPipeline):
+    def __init__(self, batch_size, num_threads, device_id, seed, image_dir):
+        super(RotatePipeline, self).__init__(batch_size, num_threads, device_id, seed, image_dir)
+        self.rotate=ops.Rotate(angle=90.0)
+    def define_graph(self):
```
Comment (jantonguirao) [inline dali/test/python/test_operator_python_function.py]: add empty line

File: dali/test/python/test_operator_python_function.py [hunk 3]
```diff
@@ -172,9 +180,56 @@ def test_python_operator_flip():
...
+        images, labels = self.load()
+        rotate=self.rotate(images)
+        return rotate
+
```
Comment (jantonguirao) [inline dali/test/python/test_operator_python_function.py]: remove one empty line

File: dali/pipeline/operators/displacement/rotate.h (hunk 1 above)
Comment (JanuszL) [reply dali/pipeline/operators/displacement/rotate.h]: Yes it is, to be aligned with all other libs (OCV or numpy) which rotates counterclockwise.

File: dali/test/python/test_operator_python_function.py (hunk 3 above)
Comment (JanuszL) [reply dali/test/python/test_operator_python_function.py]: It follows the convention that every class or function is separated by two empty lines.

File: dali/test/python/test_operator_python_function.py (hunk 2 above)
Comment (JanuszL) [reply dali/test/python/test_operator_python_function.py]: Done

**Comment by JanuszL** [comment]
!build

**Pull request merged**